builtin/mainmenu/content/dlg_package.lua

```diff
@@ -43,9 +43,14 @@ local function make_scrollbaroptions_for_scroll_container(visible_l, total_l, sc
 end
 
 
+local function escape_hypertext(text)
+	return text:gsub("\\", "\\\\"):gsub("<", "\\<"):gsub(">", "\\>")
+end
```

Comment:
https://github.com/minetest/minetest/blob/0ea1ec31fc19ea5026db9b068bb3a606a8505d0b/builtin/mainmenu/tab_about.lua#L106-L111

At this point we should really add `core.hypertext_escape` (word order to match `core.formspec_escape`)